forms/templates/forms/workflow_entries/index.html

```diff
@@ -22,7 +22,7 @@
                 <tr class="row">
                     <td>{{ response.creator.username }}</td>
                     <td>{{ response.current_state.name }}</td>
-                    {% for element in response.formelementinstance_set.all %}
+                    {% for element in response.formelemententry_set.all %}
```

Comment:
#6 marked as not-ready-to-merge. I'll merge this request then you correct that pull request for any merge issues.